src/kOS/Screen/TermWindow.cs

```diff
@@ -234,7 +234,7 @@ public void LoadTexture(string relativePath, ref Texture2D targetTexture)
             var imageLoader = new WWW("file://" + root + relativePath);
             imageLoader.LoadImageIntoTexture(targetTexture);
 
-            if (imageLoader.isDone && imageLoader.size == 0)
+            if (imageLoader.isDone && imageLoader.bytesDownloaded == 0)
```

Comment:
Fixes warning: ``WWW.size`` is obsoleted.
Apparently Unity renamed this field to use the terminology 'downloaded' (even when talking about a local ``file://`` URL).